src/SFA.DAS.FAT.Web/Views/Courses/CourseProviders.cshtml

```diff
@@ -152,44 +155,65 @@
             
             <h2 class="govuk-heading-m">@Model.TotalMessage</h2>
             
-            @if (Model.Total == 0)
+            @if (Model.TotalFiltered == 0)
             {
-                <p class="govuk-body govuk-!-margin-bottom-6">No training providers for the selected filters.</p>
-            }
-
-            <div class="das-highlight govuk-!-margin-top-0 govuk-!-margin-bottom-5">
-                <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Apprenticeship location</h2>
-                @if (Model.HasLocation)
+                if (Model.HasLocation)
                 {
-                    <p class="govuk-body govuk-!-margin-bottom-0">
-                        <span class="govuk-!-margin-right-2">@Model.Location</span>
-                        <a asp-querystring="@Model.ClearLocationLink" class="govuk-link govuk-link--no-visited-state" id="lnk-remove-location">Remove&nbsp;location</a>
-                    </p>    
+                    <p class="govuk-body">You can either:</p>
+                    <p class="govuk-body"><ul>
+                        <li><a asp-querystring="@Model.ClearLocationLink" class="govuk-link govuk-link--no-visited-state" id="lnk-remove-location-0-results">remove the location</a></li>
+                        <li><a asp-route="@RouteNames.Courses" class="govuk-link govuk-link--no-visited-state">search for another course</a></li>
+                        @if (Model.ShowEmployerDemand && configOptions.Value.EmployerDemandFeatureToggle)
```

Comment:
Might be worth refactoring later - but changing this so that the url is set in the model rather than from the injected config.